src/azure-cli/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -350,6 +350,10 @@ def set_service_properties_track2(client, parameters, delete_retention=None, del
     if not parameters.minute_metrics.enabled:
         parameters.minute_metrics.include_apis = None
 
+    if not parameters.static_website.enabled:
+        parameters.static_website.index_document = None
+        parameters.static_website.error_document404_path = None
```

Comment:
If necessary, it should be fixed from service side instead of controlling two other params' value through an switch param.

CLI should not have such customization.

Reply:
Your teams call, but yes this should be fixed on the service side of the API response.

would think that would be an internal discussion on where you want to implement the fix

Reply:
I'll contact service team later.